### DISCUSSION TITLE
Can't add Google drive link on page - Error "Sorry, link won't work on this embedded type"

### DISCUSSION THREAD

**Discussion by igorbochkarev** [outline/outline#2497] [Self Hosting]
I try to add Google Drive link on page, but permanently have an error "Sorry, link won't work on this embedded type".
My link - https://drive.google.com/drive/folders/11R8a5qjzsVHgbAxxxxxxxxxxxxxx

Can anyone help me to find the problem?

**Comment by EacAyf** [comment]
What kind of document are you trying to embed? 
Not sure about what causes this, but if I just plainly paste the link whilst typing it just works. Otherwise is your using the '/' menu, be sure to choose sheets for sheets, docs for docs etc.

**Reply by igorbochkarev** [reply]
Not Google Doc. Just Google Drive Folder.
I'm using '/' menu, select Google Drive item from list, fill the editbox with 'https://drive.google.com/drive/folders/11R8a5qjzsVHgbA......' and see an error.

**Reply by EacAyf** [reply]
I don't think you can embed a folder currently. Maybe this is something that can be added in the future kind of like this: https://dev.to/operfildoluiz/how-to-embed-google-drive-folders-74c  (although I'm not a coder of any kind, so I'm not sure how hard it is)

For the mean time, linking towards a folder does work (less elegant solution).

**Comment by aariacarterweir** [comment]
I'm having this issue for a generic document embed :/
